modules/ROOT/pages/design-documentation.adoc

```diff
@@ -0,0 +1,305 @@
+= Differences for OAS 3.0 in Rendered Documentation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+The right pane of the text editor in API Designer displays the documentation that is written into API specifications. For API specifications that are written in RAML or OAS 2.0, the ways that the documentation is rendered are identical. However, the documentation for API specifications that are written in OAS 3.0 can include multiple server definitions, as well as links and callbacks.
+
+== Documentation for Definitions of Multiple Servers
+
+OAS 3.0 lets you define servers by using the `servers` field. You can define servers globally at the top of a specification, locally for endpoints, and locally for methods. Local definitions override the definitions set at the next higher level in the specification's hierarchy. Definitions set locally for methods override definitions set at endpoints and globally. Definitions set locally for endpoints override definitions set globally.
+
+The following example API specification demonstrates different ways, indicated by ten callouts (1 - 10), that you can set server definitions. After the example API specification, the documentation for the portion of the API at each callout is illustrated with a screenshot.
```

Comment:
This is where we are supposed to use an en-dash, but, once again, AsciiDoc does not do it. So in this case, we substitute a hyphen:
```suggestion
The following example API specification demonstrates different ways, indicated by ten callouts (1-10), that you can set server definitions. After the example API specification, the documentation for the portion of the API at each callout is illustrated with a screenshot.
```
OR, to avoid the parenthetical...
```suggestion
The following example API specification demonstrates 10 different ways to set server definitions. After the example API specification, the documentation for the portion of the API at each of callouts 1 through 10 is illustrated with a screenshot.
```